frontend/src/views/SettingWorkspaceDebugLogs.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="space-y-4">
+    <div v-if="debugLogList.length > 0">
+      <div class="flex flex-row justify-between items-center textinfolabel">
+        <i18n-t keypath="debug-logs.count-of-logs">
+          <template #count> {{ debugLogList.length }} </template>
+        </i18n-t>
```

Comment:
The `i18n-t` always using for custome styles. We can use `$t("", {})` right here. FYI.
https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/frontend/src/views/AnomalyCenterDashboard.vue?L37

Reply:
TIL.